data/theVatsal3802.json

```diff
@@ -1,29 +1,68 @@
 {
-  "name": "Vatsal Adhiya",
-  "displayStatsPublic": true,
-  "type": "personal",
-  "bio": "An enthusiastic flutter developer.",
-
-  "links": [
-    {
-      "name": "Follow me on Twitter",
-      "url": "https://twitter.com/adhiya_vatsal_3",
-      "icon": "FaTwitter"
-    },
-    {
-      "name": "Let's Connect on LinkedIn",
-      "url": "https://www.linkedin.com/in/vatsal-adhiya-206479202/",
-      "icon": "FaLinkedin"
-    },
-    {
-      "name": "Follow me on Instagram",
-      "url": "https://instagram.com/thevatsal3802",
-      "icon": "FaInstagram"
-    },
-    {
-      "name": "GitHub: Will be happy to colab on open-source projects",
-      "url": "https://github.com/theVatsal3802",
-      "icon": "FaGithub"
-    }
-  ]
+    "name": "Vatsal Adhiya",
+    "displayStatsPublic": true,
+    "type": "personal",
+    "socials": [
+        {
+            "icon": "SiGmail",
+            "url": "vatsaladhiya248@gmail.com"
```

Comment:
here as well, make it `mailto:vatsaladhiya248@gmail.com`

Reply:
Done